projects/angular/src/button/_buttons.clarity.scss

```diff
@@ -380,7 +380,9 @@
   }
 
   .btn.btn-sm,
-  .btn-sm .btn {
+  .btn-sm .btn,
+  .btn.btn-small,
+  .btn-small .btn {
```

Comment:
We don't need to add `btn-small` as class in the buttons, just fix the class on the signpost-button to be `btn-sm`

Reply:
We have docs for `btn-small`, but not for `btn-sm`. Decided to retain documentation intact and all other use cases. It's still work in progress.

Reply:
It should be `btn-sm` in that `STYLES.md` file too. It's `btn-sm` in all other components and also on our website. Adding a second thing which is wrong will just make it more confusing.

Reply:
Well not all other components. Datagrid is using `btn-small`, but full migration to `btn-sm` is the right way. I'll make the change.

Reply:
Probably should be best to move renaming to another PR.

Reply:
PR for `btn-small` created #912